src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -1,5 +1,8 @@
 $font-path: "~uswds/src/fonts";
 $image-path: "~uswds/src/img";
+$teamsite-dark-blue: #112e51;
+$teamsite-light-blue: #0073b4;
+$teamsite-width: 959px;
```

Comment:
This is based on the #main-wrap width:
<img width="969" alt="screen shot 2018-10-15 at 12 31 11 pm" src="https://user-images.githubusercontent.com/16051172/46973485-3a70f600-d076-11e8-9cbd-6ae5793c11fb.png">